src/routes.py

```diff
@@ -111,6 +111,37 @@ def device_create():
 
     return jsonify(res), 200
 
+@api.route('device/edit', methods=["PATCH"])
+def device_edit():
+
+    device_data = request.get_json()
+    ID_device = device_data["DeviceID"]
+    new_area = float(device_data["NewArea"])
```

Comment:
Eu trocaria isso aqui por
```suggestion
    new_area = int(device_data["NewArea"])
```

Os dados de área são armazenados como int.

Reply:
Mas a área não poderia ser um valor quebrado?